t/venom/test_suites/security_event_autoreg/prepare/00_create_autoreg_security_event.yml

```diff
@@ -0,0 +1,36 @@
+name: Create autoreg security event
+testcases:
+- name: get_login_token
+  steps:
+  - type: get_login_token
+
+- name: create_security_event
+  steps:
+  - type: http
+    method: POST
+    url: '{{.pfserver_webadmin_url}}/api/v1/config/security_events'
+    ignore_verify_ssl: true
+    body: >-
+      {"id":"88888888","isClone":false,"isNew":true,"access_duration":"5D","actions":["autoreg","role"],"desc":"Test DHCP autoreg venom","enabled":"Y","target_category":"machine_employee","triggers":[{"dhcp_fingerprint":"{{.security_event_autoreg.node.dhcp_fingerprint_id}}"}],"whitelisted_roles":[]}
+    headers:
```

Comment:
Could you:
- Lint JSON
- Remove `isClone` and `isNew` attributes
- Use a variable for security event id, access duration and description
- Use `{{.mac_auth.roles.headless_device.id}}` as `target_category` in place of `machine_employee` (makes more sense because we are doing MAC Authentication)